ClickHouse.Client/Utility/CommandExtensions.cs

```diff
@@ -2,17 +2,27 @@
 using System.Data.Common;
 using System.Threading.Tasks;
 using ClickHouse.Client.ADO.Adapters;
+using ClickHouse.Client.ADO.Parameters;
 
 namespace ClickHouse.Client.Utility
 {
     public static class CommandExtensions
     {
-        public static DbParameter AddParameter(this DbCommand command, string parameterName, object parameterValue)
+        public static ClickHouseDbParameter AddParameter(this DbCommand command, string parameterName, object parameterValue)
```

Comment:
If this extension returns `ClickHouseDbParameter`, it should be applicable only to `ClickHouseCommand`

Reply:
done